docs/authorization/token_caching/index.rst

```diff
@@ -0,0 +1,20 @@
+
+Token Caching
```

Comment:
Maybe controversially I titled this section "token caching" rather than "token storage". This is to disambiguate with the subpages that provide references for specific versions of the same concept.

Alternative Structure
```
docs/
  - authorization/
     - token_storage/
         - token_storages.rst
         - index.rst
         - storage_adapters.rst
```

and you'd end up with pages titled "Token Storage", "Token Storages", and "Storage Adapters (Legacy)".